docs/develop/ruby/temporal-client.mdx

```diff
@@ -0,0 +1,149 @@
+---
+id: temporal-client
+title: Temporal Client - Ruby SDK
+sidebar_label: Temporal Client
+description: Learn how to create a Temporal Client, connect to Temporal Cloud, start a Workflow, and get Workflow results using the Temporal Ruby SDK.
+keywords:
+  - sdk
+  - ruby
+  - temporal client
+tags:
+  - Ruby SDK
+  - Temporal SDKs
+  - Temporal Client
+  - Certificates
+---
+
+This page shows how to do the following:
+
+- [Create a Temporal Client](#create-a-client)
+- [Connect to Temporal Cloud](#connect-to-temporal-cloud)
+- [Start a Workflow](#start-workflow)
+- [Get Workflow results](#get-workflow-results)
+
+## Create a Temporal Client {#create-a-client}
+
+A [Temporal Client](/encyclopedia/temporal-sdks#temporal-client) enables you to communicate with the [Temporal Service](/temporal-service).
+Communication with a Temporal Service includes, but isn't limited to, the following:
+
+- Starting Workflow Executions.
+- Sending Signals to Workflow Executions.
+- Sending Queries to Workflow Executions.
+- Getting the results of a Workflow Execution.
+- Providing an Activity Task Token.
+
```

Comment:
```suggestion
All applications that need to communicate directly with the Temporal Service will run inside of Temporal Clients.

Your Workers are technically Temporal Clients, because they need to poll the Temporal Service to actually do work.
Your Workflow and Activity definitions are *not* Temporal Clients -- your Worker imports these directly instead.
Each Temporal SDK includes a set of Temporal Client functions.
The `temporal` CLI is also a Temporal Client, and is generally at feature parity with the SDKs.
```